Gemfile.lock

```diff
@@ -271,3 +280,6 @@ DEPENDENCIES
   spring
   uglifier (>= 1.3.0)
   web-console (~> 2.0)
+
+BUNDLED WITH
```

Comment:
I'm not sure, I thought this is a feature of the new version of bundler and was desired.